lib/elastic_apm/transport/connection/http.rb

```diff
@@ -107,6 +107,7 @@ def open_request_in_thread(url)
           Thread.new do
             begin
               resp = post(url, body: @rd, headers: @headers.chunked.to_h)
+              @rd&.close
```

Comment:
We should probably add an `ensure` to the surrounding `begin` and put this there instead? Making sure that even if the request raises somehow, we still close the thing.

Reply:
Good spot. Done.